### ISSUE TITLE
Typeahead listener getting called multiple times?

### ISSUE THREAD

**Issue by mdathersajjad** [gwtbootstrap3/gwtbootstrap3-extras#346]
Dear Team,
Typeahead is a must have widget in this libraries and application. It serves the autocomplete purpose. The select listener is getting called multiple times even though i am adding it only once.Does any body have a fix for this.

**Comment by sjardine** [comment]
Good morning!  I am the only member of the "team" and this project is very low priority for me.  I am more than happy to look at a pull request if you would like to find the problem and submit a fix.  If not, this will likely take several months to investigate and fix.  Of course, if you would like to pay for premium support we may be able to work something out.

Also, please remember that this library is a wrapper for several JS bootstrap implementations and a fix may possibly already have been completed in the JS library.  It may just be a matter of dropping in the new typeahead.js library.
